packages/w3up-client/src/client.js

```diff
@@ -311,4 +311,46 @@ export class Client extends Base {
       proofs: options.proofs,
     })
   }
+
+  /**
+   * Removes association of a content CID with the space. Optionally, also removes
+   * association of CAR shards with space.
```

Comment:
For the very least I think we should document significant downside this function has.

```suggestion
   * association of CAR shards with space.
   *
   * ⚠️ Please be aware that all shards will be deleted even if there is another upload(s) that
   * reference same shards, which in turn could corrupt those uploads.
```